sugarcrm/shadow.ini

```diff
@@ -1,3 +1,16 @@
 extension=shadow.so
-shadow.override=imagepng@w1,finfo_file@1,ext2mime@0,mime_content_type@0,ziparchive::open@w0,ziparchive::addfile@0,tempnam@w0
-shadow.mkdir_mask=0750
```

Comment:
I'm not sure that we need to change them.
I agree to add commented lines about all settings, but don't change them from default value.